### PR TITLE
initial edits

### PR DESCRIPTION
**Pull request by dcjacks0n**
Changed title, added caption text.

### CODE REVIEW

File: _slides/04-dcjacks0n.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
 layout: slide
-title: "Welcome to our slide deck!"
+title: "Baba booey!"
 ---
 
-Caption here
+Hello, world.
```
Comment (andrew-scoppa) [inline _slides/04-dcjacks0n.md:6]: Wonderful, however have you consider removing the comma, and adding markdown

Comment (wvestal) [reply _slides/04-dcjacks0n.md:6]: another comment goes here

Comment (wpelletier-emburse) [reply _slides/04-dcjacks0n.md:6]: a great comment!

**Review by andrew-scoppa — APPROVED** [review]

**Review by wvestal — APPROVED** [review]
Exciting

**Review by wpelletier-emburse — APPROVED** [review]

**Pull request merged**